tests/test_skill_marketplace.py

```
@@ -0,0 +1,222 @@
+        assert ok is False
+        assert "failed" in msg.lower()
+
+    def test_publish_handles_error(self):
+        from datus.tools.skill_tools.skill_config import SkillConfig
+        from datus.tools.skill_tools.skill_manager import SkillManager
+
+        config = SkillConfig(marketplace_url="http://nonexistent:9999")
+        manager = SkillManager(config=config, registry=MagicMock())
+        manager.registry.get_skill_count.return_value = 0
+
+        ok, msg = manager.publish_to_marketplace("/nonexistent/path")
```

Ruff (0.14.14)

[warning] Unpacked variable `msg` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)